oslabs-beta/reactive
# Colin/update package scripts watch mode

Linked issue/ticket
No ticket number is available for this pull request.

Description
This pull request introduces improvements to our development workflow by adding watch commands specific to frontend and backend devs. Specifically:

Updated / added two watch scripts (backend:watch and frontend:watch),helps backend and frontend devs by not having to use build command.

To use: 
After running npm run build, thereafter use:
backend:watch and frontend:watch

Reproduction steps
To verify these changes:

Run npm run backend:watch – Check that TypeScript files in the backend compile automatically on save.
Run npm run frontend:watch – Check that Webpack re-bundles frontend files as they are saved.
Optional: Test npm run build to see that it still compiles and bundles both backend and frontend code.
Checklist
- [ x] I've followed the [Contributing guidelines](https://github.com/oslabs-beta/.github/blob/main/docs/CONTRIBUTING.md)
- [ x] I've titled my PR according to the [Conventional Commits spec](https://conventionalcommits.org)
- [x ] I've linted, tested, and commented my code
- [ x] I've updated documentation (if appropriate)
